docs/readmes/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -5,11 +5,15 @@ hide_title: true
 ---
 # Build and install a magma package in AGW
 
-**Description:** Purpose of this document is describe how to build and install a magma package in AGW.
+## Description
 
-**Environment:** AGW
+Purpose of this document is describe how to build and install a magma package in AGW.
```

Comment:
`Purpose of this document is _to_ describe...`

Reply:
I would prefer "The purpose is..." too

Reply:
changed